Makefile

```diff
@@ -16,6 +16,13 @@ LDFLAGS += -X "github.com/mattermost/focalboard/server/model.BuildNumber=$(BUILD
 LDFLAGS += -X "github.com/mattermost/focalboard/server/model.BuildDate=$(BUILD_DATE)"
 LDFLAGS += -X "github.com/mattermost/focalboard/server/model.BuildHash=$(BUILD_HASH)"
 
+# MAC cpu architecture
+ifeq ($(shell uname -m),arm64)
+	MAC_GO_ARCH := arm64
+else
+	MAC_GO_ARCH := amd64
```

Comment:
ci server reporting `$(shell uname -m)` -> x86_64 which doesn't seem to be valid.